tool/ci_script_shared.sh

```diff
@@ -16,11 +32,56 @@ function ci_package () {
 
         # Run the formatter on all the dart files to make sure everything's linted.
         dart format --output none --set-exit-if-changed .
+        
+        flutter config --enable-native-assets
+
+        echo "$ flutter config --list 1"
```

Comment:
I remember we added oodles of echos for debugging CI, do we need them now?

Reply:
finally removed!